sdk/spring/azure-spring-starter-storage/src/main/resources/META-INF/spring.provides

```diff
@@ -1 +1 @@
-provides: spring-cloud-azure-context, spring-cloud-azure-autoconfigure
```

Comment:
Hi, @saragluna , 
Maybe we should delete all `spring.provides`?
Refs: https://github.com/spring-projects/spring-boot/issues/12435

Reply:
Yes, we should remove them. Let's create an issue to track the task.

Reply:
@saragluna , I created a PR: https://github.com/Azure/azure-sdk-for-java/pull/16618